boards/e104-bt5010a-tb/Makefile.include

```diff
@@ -1 +1,4 @@
+# setup JLink for flashing
+JLINK_DEVICE = nrf52810_xxaa
```

Comment:
Would it make sense to set this in `cpu/` instead?

Reply:
> Would it make sense to set this in `cpu/` instead so we don't need to have this for every board?

Here? https://github.com/RIOT-OS/RIOT/blob/master/cpu/nrf52/Makefile.include#L17-L36
But I would probably replicate the IF block, so it is separate from the other definitions.


Something like this:
```makefile
# Set the target for the JLink programmer for flashing.
# Boards that do not use a JLink ignore the flag.
ifneq (,$(filter nrf52805xxaa,$(CPU_MODEL)))
  JLINK_DEVICE = nrf52805_xxaa
endif
ifneq (,$(filter nrf52810xxaa,$(CPU_MODEL)))
  JLINK_DEVICE = nrf52810_xxaa
endif
ifneq (,$(filter nrf52811xxaa,$(CPU_MODEL)))
  JLINK_DEVICE = nrf52811_xxaa
endif
ifneq (,$(filter nrf52820xxaa,$(CPU_MODEL)))
  JLINK_DEVICE = nrf52820_xxaa
endif
ifneq (,$(filter nrf52832xxaa,$(CPU_MODEL)))
  JLINK_DEVICE = nrf52832_xxaa
endif
ifneq (,$(filter nrf52833xxaa,$(CPU_MODEL)))
  JLINK_DEVICE = nrf52833_xxaa
endif
ifneq (,$(filter nrf52840xxaa,$(CPU_MODEL)))
  JLINK_DEVICE = nrf52840_xxaa
endif
```

Reply:
I tried it out and it does work. Before comitting it I'd like to wait for your feedback whether or not adding it to a separate IF-block or adding it to the existing block.

Reply:
That looks good to me!